src/ui/public/filter_bar/filter_bar.less

```diff
@@ -76,6 +95,34 @@ filter-bar {
     line-height: 1.5;
   }
 
+  .action-show {
+    position: absolute;
+    right: 10px;
+    bottom: 0;
+  }
+
+  .filter-nav-link__icon {
+    display: inline;
+    position: absolute;
+    top: 10px;
+    right: 15px;
+    opacity: 0.35;
```

Comment:
Unfortunately/fortunately we have to be AA contrast compliant for accessibility, the lightest we can go here would be `opacity: 0.75`